[OpenAPI 3 Emitter] Reuse Operation IDs With Identical Values In @OperationId Decorators For Shared Routes (#6157)
microsoft/typespec: .chronus/changes/wanl-opid-2025-1-26-11-56-11.md

---
changeKind: feature
packages:
  - "@typespec/openapi3"
---

> Reuse operation IDs when all `@operationId` decorators share the same value, which improve the usability for client code generation to leverage the shared operation ID.

Reviewer: ```suggestion
Shared operations operationId can now be set if they all share the same value provided by `@operationId` 
```
